chxtools/handlers2.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 import h5py
 from pims import FramesSequence, Frame
-from databroker import db
+from databroker import Broker
+from .handlers import LazyEigerHandler
```

Comment:
eventually, should be 
`from eiger_io.fs_handler import EigerHandler`
for `PIMS` version
or
`from eiger_io.fs_handler_dask import EigerDaskHandler`
for `dask` version.

But let's keep it as is for now. There are some things I need to check back with Yugang on.

Just to summarize what needs to be fixed with this:
1. The handlers that treat the `AD_EIGER` and `AD_EIGER2` specs should be combined into one. The only difference between the two is that the keyword argument is a different name. (Other that that API is same as far as I can tell and I've already tested this on all versions of EIGER files, of which there are four in CHX currently)
2. The `EigerImages` `pims` subclass (which is created by the handler per event) contains a lot of metadata which is obtained by reading from the hdf5 file directly. We should discourage that and try to make sure CHX accesses this data directly from `metadatastore`. Most of this data should be in a descriptor `configuration`. I want to discuss with Yugang and try my best to make sure we weed this out, and make a long term plan for deprecating this use.

Reply:
PR here: https://github.com/NSLS-II-CHX/eiger-io/pull/8